cfg/migrate/awscsm.go

```diff
@@ -9,10 +9,10 @@ func init() {
 	AddRule(AwsCsmRule)
 }
 
-//  [agent]
-//-   enable_csm_reflection = true
-//    [[inputs.awscsm_listener]]
-//+     data_format = "aws_csm"
+//		[agent]
```

Comment:
There are a lot of extraneous formatting changes, and not all of them are warranted, like this one. What did you run to format these files?

Reply:
I believe i did `Make fmt`

Reply:
Apologies, i just realised my editor ran auto format